packages/terra-table/src/SingleSelectableRows.jsx

```diff
@@ -60,10 +65,26 @@ class SingleSelectableRows extends React.Component {
     };
   }
 
-  newPropsForRow(row, index, onClick) {
+  wrappedOnKeyDownForRow(row, index) {
+    const initialOnKeyDown = this.props.onKeyDown;
+
+    return (event) => {
+      if (event.nativeEvent.keyCode === Number(13)) {
+        if (row.props.isSelectable !== false && this.shouldHandleSelection(index)) {
+          this.handleSelection(event, index);
+        }
+
+        if (initialOnKeyDown) {
+          initialOnKeyDown(event);
+        }
```

Comment:
We always want to execute the initialOnKeyDown function, even if the key pressed isn't Enter. We need to break this if-block out of the key check.